src/main/java/atmosfears/AtmosFearsBE/controller/HelloController.java

```diff
@@ -1,14 +1,18 @@
 package atmosfears.AtmosFearsBE.controller;
 
+import atmosfears.AtmosFearsBE.service.AirParticulatesService;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 public class HelloController {
 
+    @Autowired
+    private AirParticulatesService airParticulatesService;
     @GetMapping("/")
     public String index() {
-        return "Greetings from Spring Boot!";
+        return "Greetings from Spring Boot! <br> Repository findAll result: " + airParticulatesService.findAll();
```

Comment:
You don't have to use the `@Autowired` decorator. It's even not recommended and the better approach is to use constructor injection instead.

You can replace selected lines of code with the following code:

```ts
private final AirParticulatesService airParticulatesService;

private HelloController(AirParticulatesService airParticulatesService) {
    this.airParticulatesService = airParticulatesService;
}
```